packages/shared/src/features/boost/FeedCardContext.tsx

```diff
@@ -0,0 +1,11 @@
+import { createContext, useContext } from 'react';
+
+type FeedCardContextData = {
+  isBoostedReach: boolean;
+};
+
+export const FeedCardContext = createContext<FeedCardContextData>({
```

Comment:
I don't think this should be under the boost feature, this is more a generic post thing no?
I would also just accept any partial post for future proofing.

And didn't we want to use kickass context?

Reply:
> I don't think this should be under the boost feature, this is more a generic post thing no?

Yeah this is not specifically just for boost feature, it just so happened that it fits the case now. I didn't add the post object yet as there is no use for now.

Yeah, but we found some concerns hence we had to revert back to just this.

Reply:
But then let's move it out of the boost feature folder?
And just make it value <Optional<Post>> no?

Reply:
Not sure what I understand the <Optional> part.